specs/protocol/isthmus/derivation.md

```diff
@@ -17,11 +24,261 @@ The Isthumus hardfork activation block contains the following transactions, in t
 - L1 Attributes Transaction
 - User deposits from L1
 - Network Upgrade Transactions
+  - L1Block deployment
+  - GasPriceOracle deployment
+  - Operator Fee vault deployment
+  - Update L1Block Proxy ERC-1967 Implementation
+  - Update GasPriceOracle Proxy ERC-1967 Implementation
+  - Update Operator Fee vault Proxy ERC-1967 Implementation
+  - GasPriceOracle Enable Isthmus
   - EIP-2935 Contract Deployment
 
 To not modify or interrupt the system behavior around gas computation, this block will not include any sequenced
 transactions by setting `noTxPool: true`.
 
+## L1Block deployment
+
+The `L1Block` contract is upgraded to support the Isthmus operator fee feature.
+
+A deposit transaction is derived with the following attributes:
+
+- `from`: `0x4210000000000000000000000000000000000003`
+- `to`: `null`
+- `mint`: `0`
+- `value`: `0`
+- `gasLimit`: `425,000`
+- `data`: `0x60806040523480156100105...` ([full bytecode](../../static/bytecode/isthmus-l1-block-deployment.txt))
+- `sourceHash`: `0x3b2d0821ca2411ad5cd3595804d1213d15737188ae4cbd58aa19c821a6c211bf`,
+  computed with the "Upgrade-deposited" type, with `intent = "Isthmus: L1 Block Deployment"
+
+This results in the Isthmus L1Block contract being deployed to `0xFf256497D61dcd71a9e9Ff43967C13fdE1F72D12`, to verify:
+
+```bash
+cast compute-address --nonce=0 0x4210000000000000000000000000000000000003
+Computed Address: 0xFf256497D61dcd71a9e9Ff43967C13fdE1F72D12
+```
+
+Verify `sourceHash`:
+
+```bash
+cast keccak $(cast concat-hex 0x0000000000000000000000000000000000000000000000000000000000000002 $(cast keccak "Isthmus: L1 Block Deployment"))
+# 0x3b2d0821ca2411ad5cd3595804d1213d15737188ae4cbd58aa19c821a6c211bf
+```
+
+Verify `data`:
+
+```bash
+git checkout TODO
+make build-contracts
+jq -r ".bytecode.object" packages/contracts-bedrock/forge-artifacts/L1Block.sol/L1Block.json
+```
+
+This transaction MUST deploy a contract with the following code hash
+`TODO`.
```

Comment:
Also missing `TODO` here

Reply:
The code hash need to be from a contract with a release version